src/main/java/org/tndata/android/compass/CompassApplication.java

```diff
@@ -133,7 +135,9 @@ public void updateAction(Action action) {
     @Override
     public void onCreate(){
         super.onCreate();
-        Fabric.with(this, new Crashlytics());
+        if(!BuildConfig.DEBUG) {
```

Comment:
@bradmontgomery @damianurbanczyk do we really want to have crash reporting only in debug builds? I think it would be much more useful if we gathered that information in release builds as well to prioritise fixes on problems that the bigger section of the general public is experiencing.